Extensions/TileMap/pixi-tilemap/dist/pixi-tilemap.umd.js

```diff
@@ -77,7 +77,7 @@ this.PIXI.tilemap = this.PIXI.tilemap || {};
          * The width/height of each texture tile in a {@link TEXTILE_DIMEN}. This is 1024px by default.
          *
          * This should fit all tile base-textures; otherwise, {@link TextileResource} may fail to correctly
-         * upload the textures togther in a tiled fashion.
+         * upload the textures together in a tiled fashion.
```

Comment:
This file is an external library.
You can report these typo to https://github.com/pixijs/tilemap if you want.

Reply:
I see! I'll revert for now and fix it in an external library!

Reply:
I opened the PR! pixijs/tilemap#152